app/(main)/league/[leagueId]/entry/Entries.interface.ts

```diff
@@ -8,13 +8,13 @@ export interface IEntry {
   name: string;
   user: IUser;
   league: ILeague;
-  selectedTeams: INFLTeam[];
+  selectedTeams: INFLTeam['teamName'][];
   eliminated: boolean;
 }
 
 export interface IEntryProps {
   name: string;
   user: IUser['id'];
   league: ILeague['leagueId'];
-  selectedTeams?: INFLTeam[];
+  selectedTeams?: INFLTeam['teamName'][];
```

Comment:
Why is teamName in quotation marks here? Is it asserting it's type as a string? Or is it going to statically be 'teamName'?

Reply:
We're directly using a key from this Interface.